aesara/tensor/basic.py

```diff
@@ -1613,6 +1613,24 @@ def full(shape, fill_value, dtype=None):
     return alloc(fill_value, *shape)
 
 
+def full_like(
+        a: TensorVariable,
+        fill_value: Union[TensorVariable, int, float],
+        dtype=None,
```

Comment:
Couldn't find a good type hint for `dtype`, suggestions?

Reply:
`Union[str, np.generic, np.dtype]` should be valid.